static_src/test/functional/expressive_empty_state.spec.js

```diff
@@ -0,0 +1,40 @@
+
+import http from 'axios';
+import dedent from 'dedent';
+
+import EmptyElement from './pageobjects/empty.element';
+import OrgQuicklookElement from './pageobjects/org_quicklook.element';
+
+describe('Expressive empty state', function () {
+  let appsEmptyState;
+
+  it('navigates to page', function () {
+    browser.url('/');
+  });
+
+  it('changes to app emtpy state', function(done) {
+    const server = browser.getUrl().split('/')[2];
+
+    const result = browser.executeAsync(function(url, done) {
+      var req = fetch(`http://${url}/_admin/api/route/space-summary`, {
```

Comment:
I would avoid using `fetch` here, I'm not sure it allows a JSON object for the body (I think you need to pass it a binary type like `Blob). I tried using axios here and it works fine.